README.md

```diff
@@ -139,6 +139,8 @@ The structure of the configuration is described below:
 | `cluster.advertise` | `EMITTER_CLUSTER_ADVERTISE` | The address and port to advertise inter-node communication network. This is used for nat traversal. |
 | `cluster.seed` | `EMITTER_CLUSTER_SEED` | The seed address (or a domain name) for cluster join. |
 | `cluster.passphrase` | `EMITTER_CLUSTER_PASSPHRASE` | Passphrase is used to initialize the primary encryption key in a keyring. This key is used for encrypting all the gossip messages (message-level encryption). |
+| `storage.provider` | Not Support |  This property represents the publishers publish message storage mode. there are two kinds of can use, they are respectively `inmemory` and `ssd`, defaults to the former. |
```

Comment:
The environment variable should be supported. It should be `EMITTER_STORAGE_PROVIDER` and `EMITTER_STORAGE_CONFIG` respectively (the config you just provide the JSON)

Reply:
I don't know the knowledge of docker, so I didn't supplement the configuration of environment variables before. I will close this pr !